docs/bplan_api.md

```diff
@@ -2,169 +2,308 @@
 
 mein.berlin provides an external REST API to create and manage BPLAN projects.
 
+## Background
+
+Bplans define binding regulations for urban planning, specifying land
+use (e.g. housing, offices, green spaces) and building parameters like height
+and footprint. Citizens can participate in the process and provide feedback
+on the plans. As the central participation platform of the city of Berlin, we want
+to show all Bplans which allow participation in our project overview. Up until the beginning of 2025
+meinBerlin also provides the means for the digital participation. In the future this will be provided by
+the Diplan platform.
+
+### Integration with Imperia (legacy)
+
+Imperia is a CMS used by the public administration and is responsible for displaying Bplans and the participation form
+on berlin.de. Imperia uses this API to publish and unpublish Bplans on meinBerlin depending on whether participation is
+currently possible. When creating or publishing a Bplan, the meinBerlin API returns an embed code (iframe) which then is
+embedded in the Bplan page on berlin.de to allow citizen to submit their feedback. Imperia only provides basic
+information about the Bplan to meinBerlin, therefore meinBerlin will fetch the location and the district from the
+Bplan map.
```

Comment:
from what I read in the wiki and understood from the test cases, initiators and orga group users can create bplans, beside Imperia, correct? where in the dashboard? Or only from django-admin?

Reply:
yes they can be created in the dashboard as well

Reply:
added a note